src/test/java/racinggame/model/CarModelTest.java

```diff
@@ -0,0 +1,108 @@
+package racinggame.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import com.sun.media.sound.InvalidFormatException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class CarModelTest {
+
+  Car defaultCar;
+  @BeforeEach
```

Comment:
```suggestion
  Car defaultCar;
  
  @BeforeEach
```

한칸 띄어준다면 읽기 더 편할 것 같아요 👍